File: DisasterServer/Maps/DesertTown.cs
using DisasterServer.Session;

namespace DisasterServer.Maps;

public class DesertTown : Map
{
	public override void Init(Server server)
	{
		SetTime(server, 180);
		base.Init(server);
	}

	public override void Tick(Server server)
	{
		base.Tick(server);
	}

	protected override int GetRingSpawnCount()
	{
		return 25;
	}
}

File: disasterserver/Maps/DesertTown.cpp
#include "DesertTown.hpp"

#include "Server.hpp"

using namespace DisasterServer;
using namespace DisasterServer::Maps;

DesertTown::DesertTown(Server &server) : Map(server, "Desert Town", 1, 30) {}

void DesertTown::init(GameState& game)
{
}

void DesertTown::tick() {

}

void DesertTown::handle(Client&, Packet&) {

}

void DesertTown::left(Client&) {

}

DisasterServer::MapProperties DesertTown::getMapProperties() const {
    return MapProperties();
}
